build.sc

```diff
@@ -454,6 +455,11 @@ object contrib extends MillModule {
     }
    }
 
+  object proguard extends MillModule {
+    override def moduleDeps = Seq(scalalib)
+    def ivyDeps = Agg(Deps.proguard, Deps.osLib)
```

Comment:
Why we depend on `Deps.proguard` at build time?

Reply:
Will remove